lib/aio/github.py

```diff
@@ -66,8 +69,19 @@ def __init__(self, name: str, config: JsonObject) -> None:
 
     async def __aenter__(self) -> Self:
         headers = {}
-        # token is mandatory if `post = true`
-        if token := get_str(self.config, 'token', *((None,) if self.dry_run else ())):
```

Comment:
> The most obvious fix (adjusting the config file to `post=false`) is not an option at present because our CI deployment doesn't contain an explicit `forge.github.post = true`

This is easy to change -- if that makes things any simpler, go right ahead.